analyses/pypi/generate_pylist.py

```diff
@@ -0,0 +1,30 @@
+# This chunk is used in generating pylist.json from requirements.txt.
```

Comment:
I see that dependency generator for python has been kept as a normal file, what is the distribution strategy? I mean how this file would be delivered along with the cli?

Probably you should consider using `go generate`. more info: https://blog.golang.org/generate

Reply:
i am of the idea that this file would be converted to binary by `go build`

Reply:
This file won't be included into the binary unless you use `go generate` techniques.

Reply:
Addressed this by converting go string to python file